src/Extension/AssetTrait.php

```diff
@@ -180,16 +180,16 @@ private function normalizeAsset(FileAssetInterface $asset)
             return;
         }
 
-        $file = $this->getWebDirectory()->getFile($asset->getPath());
+        $app = $this->getContainer();
+
+        $file = $app['filesystem']->getFile("extensions_assets://{$this->getId()}/$path");
```

Comment:
Thinking about it a bit more, maybe that (lowercase) should be the standard. In that way there is no difference between path patterns for bundles and managed extensions. Make sense?